service-monitor/src/main/java/com/yahoo/vespa/service/monitor/internal/health/ApplicationHealthMonitor.java

```diff
@@ -6,35 +6,36 @@
 import com.yahoo.config.model.api.PortInfo;
 import com.yahoo.config.model.api.ServiceInfo;
 import com.yahoo.config.provision.ApplicationId;
+import com.yahoo.config.provision.HostName;
 import com.yahoo.vespa.applicationmodel.ClusterId;
 import com.yahoo.vespa.applicationmodel.ConfigId;
 import com.yahoo.vespa.applicationmodel.ServiceStatus;
 import com.yahoo.vespa.applicationmodel.ServiceType;
-import com.yahoo.vespa.athenz.identity.ServiceIdentityProvider;
 import com.yahoo.vespa.service.monitor.ServiceStatusProvider;
 import com.yahoo.vespa.service.monitor.application.ApplicationInstanceGenerator;
 import com.yahoo.vespa.service.monitor.internal.ServiceId;
 
-import java.net.URL;
-import java.util.Collection;
+import java.util.Arrays;
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
 import java.util.Optional;
 
-import static com.yahoo.yolean.Exceptions.uncheck;
-
 /**
  * Responsible for monitoring a whole application using /state/v1/health.
  *
  * @author hakon
  */
 public class ApplicationHealthMonitor implements ServiceStatusProvider, AutoCloseable {
+    public static final String PORT_TAG_STATE = "STATE";
+    public static final String PORT_TAG_HTTP = "HTTP";
+    /** Port tags implying /state/v1/health is served */
+    public static final List<String> PORT_TAGS_HEALTH = Arrays.asList(PORT_TAG_HTTP, PORT_TAG_STATE);
```

Comment:
Wrap this in `Collections.unmodifiableList()`?

Reply:
Done